src/RpcClient/TransactionManager.cs

```diff
@@ -84,8 +84,6 @@ public TransactionManager MakeTransaction(byte[] script, TransactionAttribute[]
                 long remainder = Tx.SystemFee % d;
                 if (remainder > 0)
                     Tx.SystemFee += d - remainder;
-                else if (remainder < 0)
```

Comment:
Why this change?

Reply:
![image](https://user-images.githubusercontent.com/4393234/78644823-17c94180-78e9-11ea-9545-47b9ef8f5422.png)
This case can never be met for the `Tx.SystemFee > 0`.